tests/internals.rs

```diff
@@ -1698,10 +1706,10 @@ fn test_deploy_account_revert() {
         .storage_initial_values_mut()
         .extend(state_after.cache_mut().storage_initial_values_mut().clone());
 
-    // Set contract class cache
-    state_reverted
-        .set_contract_classes(state_after.contract_classes().clone())
-        .unwrap();
+    // // Set contract class cache
+    // state_reverted
+    //     .set_contract_classes(state_after.contract_class_cache().clone())
+    //     .unwrap();
```

Comment:
Why was this commented?

Reply:
The setter has been removed. Before that it was already useless because the contract class cache was already set in the constructor. Removed.